kitchen/kitchen.run-hook.sh

```diff
@@ -37,6 +37,10 @@ if [ -e "${SCRIPT}" ]; then
         TOKEN=$(curl -X PUT "http://169.254.169.254/latest/api/token" -H "X-aws-ec2-metadata-token-ttl-seconds: 21600") \
         && curl -H "X-aws-ec2-metadata-token: $TOKEN" -v http://169.254.169.254/latest/meta-data/instance-id
       ')
+      echo "Install libxcrypt-compat dmidecode package by using SSH key: ${KITCHEN_SSH_KEY_PATH}"
```

Comment:
If this is something required only by AL2023, let's add a comment saying that. 
If this is not, why adding it now?

Reply:
Since this is in test code, can we address this later?